docs/topics/gradle/gradle-configure-project.md

```diff
@@ -635,9 +635,19 @@ Learn more about [Kotlin Multiplatform for different platforms](https://www.jetb
 
 It's recommended to use Android Studio for creating Android applications. [Learn how to use the Android Gradle plugin](https://developer.android.com/studio/releases/gradle-plugin).
 
-## Targeting JavaScript
+## Targeting the web
 
-When targeting JavaScript, use the `kotlin-multiplatform` plugin as well. [Learn more about setting up a Kotlin/JS project](js-project-setup.md)
+Kotlin Multiplatform (KMP) offers two approaches for web development:
+
+* JavaScript-based (using the Kotlin/JS compiler)
+* WebAssembly-based (using the Kotlin/Wasm compiler)
+
+Both approaches use the Kotlin Multiplatform plugin but target different environments.
```

Comment:
I would say:
> Both approaches use the Kotlin Multiplatform plugin but target different use cases.

Because the environment (sometimes) is the same.